npm_deps.bzl

```diff
@@ -0,0 +1,9 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "npm_install")
+
+def npm_deps():
+
+    npm_install(
```

Comment:
is this file worth the extra indirection? why not just call npm_install in the WORKSPACE?

Reply:
I guess not. I will just call It in rules_webpack_dependencies macro

Reply:
oh, i just realized it would be problematic to call in the macro. i will just call it in the workspace.